docs/ops/pooling/AdaptiveMaxPool_8.md

```diff
@@ -0,0 +1,81 @@
+## AdaptiveMaxPool<a name="AdaptiveMaxPool"></a> {#openvino_docs_ops_pooling_AdaptiveMaxPool_8}
+
+**Versioned name**: *AdaptiveMaxPool-8*
+
+**Category**: *Pooling*
+
+**Short description**: Applies max pooling with adaptive kernel size over the input.
+
+**Detailed description**: This operation calculates the output based on the input and `output_size`.
+The kernel dimensions are calculated using the following formulae for the `NCDHW` input case:
+
+\f[
+\begin{array}{lcl}
+d_{start}&=&floor(i*D_{in}/D_{out})\\
+d_{end}&=&ceil((i+1)*D_{in}/D_{out})\\
+h_{start}&=&floor(j*H_{in}/H_{out})\\
+h_{end}&=&ceil((j+1)*H_{in}/H_{out})\\
+w_{start}&=&floor(k*W_{in}/W_{out})\\
+w_{end}&=&ceil((k+1)*W_{in}/W_{out})
+\end{array}
+\f]
```

Comment:
the formula looks not good. Could you please re-write it?

Reply:
How you suggest to improve it? I added spaces for better readability